### PR TITLE
Boban/5134 focus on errors

### LINKED ISSUE #5134: Focus on error message when an invalid test result is uploaded
## Area or Component
Test results bulk upload

## Quick Summary
Error message that appears does not gain focus when an invalid csv is uploaded.

### How to Replicate
Upload an invalid results csv. An error message will appear but it will not gain focus.

## Accessibility Violation

[why is this inaccessible? include deque link if available.]: #

## Action Requested
Ensure the error message gains focus once it appears on the page.

## Source Code (optional)

[which file or line is the issue attached to?]: #

### PR DESCRIPTION
**Pull request by BobanL**
# FRONTEND PULL REQUEST

## Related Issue

- Resolves #5134 

## Changes Proposed

- When an error occurs, focus on the error.
- Add aria-invalid to file input
- Add a link to the error message back down to the file input

## Testing

- Force an validation error locally and make sure focus is set on [dev3](https://dev3.simplereport.gov/app)

### CODE REVIEW

**Review by fzhao99 — DISMISSED** [review]
lgtm!

File: frontend/src/app/patients/UploadPatients.tsx [hunk 1]
```diff
@@ -209,7 +217,7 @@ const UploadPatients = () => {
     return (
       <div>
         {errorMessage.body && (
-          <div className="usa-alert usa-alert--error maxw-560">
+          <div className="usa-alert usa-alert--error maxw-560" tabIndex={-1}>
```
Comment (johanna-skylight) [inline frontend/src/app/patients/UploadPatients.tsx:220]: Why is the tabIndex -1 necessary?

File: frontend/src/app/testResults/uploads/Uploads.tsx [hunk 2]
```diff
@@ -40,6 +40,14 @@ const Uploads = () => {
   >([]);
   const [errorMessageText, setErrorMessageText] = useState<string | null>(null);
 
+  useEffect(() => {
+    if (errorMessageText) {
+      (
+        document.getElementsByClassName("usa-alert--error")[0] as HTMLDivElement
+      ).focus();
```
Comment (johanna-skylight) [inline frontend/src/app/testResults/uploads/Uploads.tsx:47]: The focus should be place in the form input

File: frontend/src/app/patients/UploadPatients.tsx (hunk 1 above)
Comment (BobanL) [reply frontend/src/app/patients/UploadPatients.tsx:220]: `tabIndex` was added to make the div focusable and then the -1 is to only allow it be focused via `.focus()`.

Comment (johanna-skylight) [reply frontend/src/app/patients/UploadPatients.tsx:220]: the focus should be place in the file input not the alert element and I think this alert is missing the role="alert"

Comment (BobanL) [reply frontend/src/app/patients/UploadPatients.tsx:220]: The ticket says `Ensure the error message gains focus once it appears on the page.`. We can talk to @emyl3 to confirm with them when we all get back on Tuesday. 
I can add role=alert on that one though.

Comment (johanna-skylight) [reply frontend/src/app/patients/UploadPatients.tsx]: yeah, makes sense... I might be wrong

**Review by mehansen — DISMISSED** [review]
looks and behaves right to me per my understanding of the ticket

**Comment by BobanL** [comment]
Ready for re-review after adding @johanna-skylight's comment around adding aria-invalid and a link back to the input file!

**Review by johanna-skylight — APPROVED** [review]
Looks good :). Thank you for the very polished implementation!!!

**Review by emyl3 — APPROVED** [review]
Checked the axe plugin and also checked with VoiceOver to ensure the errors get focus properly and it all works as intended!

**Pull request merged**